packages/p2p/src/components/my-ads/create-ad-add-payment-method-modal.jsx

```diff
@@ -49,29 +58,43 @@ const CreateAdAddPaymentMethodModal = () => {
                 is_modal_open={my_ads_store.should_show_add_payment_method_modal}
                 page_header_className='buy-sell__modal-header'
                 page_header_text={localize('Add payment method')}
-                pageHeaderReturnFn={() => my_ads_store.setShouldShowAddPaymentMethodModal(false)}
+                pageHeaderReturnFn={onCancel}
             >
-                <AddPaymentMethod should_show_page_return={false} should_show_separated_footer={true} />
+                <AddPaymentMethod
+                    formik_ref={formik_ref}
+                    should_show_page_return={false}
+                    should_show_separated_footer={true}
+                />
             </MobileFullPageModal>
         );
     }
 
     return (
         <Modal
             className='p2p-my-ads__modal-error'
-            has_close_icon={false}
+            has_close_icon
```

Comment:
Added close icon because it was showed in Figma design:
![Screenshot 2022-05-26 at 1 19 04 PM](https://user-images.githubusercontent.com/103016120/170426877-7eade6b2-fc30-44ea-8176-1b44c8aa4058.png)